src/components/app/Homepage.js

```diff
@@ -4,12 +4,33 @@ import Heading from '../core/Heading';
 import Page from '../core/Page';
 import Header from '../app/Header';
 import Footer from '../app/Footer';
+import Hero from '../core/Hero';
 import Features from '../app/Features';
 import DogFinder from '../app/DogFinderContainer';
+import { Inverted } from '../../ThemeProvider';
 
-const Layout = () => (
+const BACKGROUND = `linear-gradient(to bottom, rgba(44,35,28,0.7), rgba(0,0,0,0)),
```

Comment:
Why extracting it if it only being used once?

Reply:
Doesn't matter ;-) At the time of writing I thought it looks better. Maybe Prettier formats the  code in a weird way with a const ;-)

Reply:
It does IMO since it communicates that it may be reused somewhere in code and also introduces an indirection when code being read. I wouldn’t use it if not necessary.

Reply:
I'd say code reuse is a minor reason to extract some code, but I can inline it: I don't see any difference in this case ;-)